unittests/Frontend/IsFeatureEnabledTests.cpp

```diff
@@ -0,0 +1,641 @@
+//===-- IsFeatureEnabledTests.cpp -------------------------------*- C++ -*-===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2025 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+#include "FeatureParsingTest.h"
+#include <map>
+
+using namespace swift;
+
+namespace {
+
+static const FeatureWrapper baselineF(Feature::AsyncAwait);
+static const FeatureWrapper upcomingF(Feature::DynamicActorIsolation);
+static const FeatureWrapper adoptableUpcomingF(Feature::ExistentialAny);
+static const FeatureWrapper experimentalF(Feature::NamedOpaqueTypes);
```

Comment:
An experimental feature is only available in asserts toolchains, like the one CI tests by default, but it will not be available in non asserts toolchains, like the ones that nightly does (I think). This will probably fail on non asserts toolchains.

Reply:
THANK YOU. I will swap it for an always available experimental feature.